kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection/subPages/PreviewSelectedResources/PreviewContent.vue

```diff
@@ -0,0 +1,268 @@
+        type: Object,
+        required: true,
+      },
+      ancestors: {
+        type: Array,
+        required: false,
+        default: () => [],
+      },
+      isSelected: {
+        type: Boolean,
+        required: true,
+      },
+      questions: {
+        type: Array,
+        required: false,
+        default: () => [],
+      },
+    },
+    data() {
+      return {
+        selectedQuestionIndex: 0,
+      };
+    },
+    computed: {
+      channelsLink() {
+        return {
+          name: PageNames.LESSON_SELECT_RESOURCES_INDEX,
+        };
+      },
+      isExercise() {
+        return this.content.kind === ContentNodeKinds.EXERCISE;
+      },
+      selectedQuestion() {
+        if (this.isExercise) {
+          return this.questions[this.selectedQuestionIndex];
+        }
+        return '';
+      },
+      licenseName() {
+        return licenseLongName(this.content.license_name);
+      },
+      content() {
+        return this.currentContentNode;
+      },
+      description() {
+        if (this.content && this.content.description) {
+          const md = new markdownIt('zero', { breaks: true });
+          return md.render(this.content.description);
+        }
+
+        return undefined;
+      },
+      learningActivities() {
+        if (this.content.learning_activities) {
+          return this.content.learning_activities;
+        }
+        return [];
+      },
+    },
+    methods: {
+      topicsLink(topicId) {
+        const { params, query } = this.$route;
+
+        return {
+          name: PageNames.LESSON_SELECT_RESOURCES_TOPIC_TREE,
+          params: params,
+          query: {
+            ...query,
+            topicId,
+          },
+        };
+      },
+      questionLabel(questionIndex) {
+        if (!this.isExercise) {
+          return '';
+        }
+        const questionNumber = questionIndex + 1;
+        return this.coreString('questionNumberLabel', { questionNumber });
+      },
+      addResource() {
+        this.$emit('addResource', this.content);
+      },
+      removeResource() {
+        this.$emit('removeResource', this.content);
+      },
+      getTime(seconds) {
+        return this.$tr('minutes', { value: Math.floor(seconds / 60) });
+      },
+    },
+    $trs: {
+      minutes: {
+        message: '{value, number, integer} {value, plural, one {minute} other {minutes}}',
+        context:
+          'Indicates time spent by learner on a specific activity. Only translate minute/minutes.',
+      },
+    },
+  };
+
+</script>
+
+
+<style lang="scss" scoped>
+
+  .license-detail-style {
+    margin: 30px 0 32px;
+  }
+
+  /deep/ .content-renderer {
+    position: relative;
+    top: -40px;
+    max-height: 500px;
+  }
+
+  .channel-header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .title-class {
+    position: relative;
+    top: -30px;
+  }
+
+  .align-breadcrumbs {
+    position: relative;
+    top: -35px;
```

Comment:
Idem

Reply:
If the motivation behind all these negative relative values is to remove this margin from the "select from channels" label. Then it will be better to remove the margins from the h6 node, so the rest will be placed inmediately.

![image](https://github.com/user-attachments/assets/a1e4ee8e-1b36-429c-b018-6e763581ef51)